test/functional/apps/visualize/_region_map.js

```diff
@@ -82,8 +82,12 @@ export default function ({ getService, getPageObjects }) {
 
       it('should contain a dropdown with the default road_map base layer as an option',
         async () => {
-          const roadMapExists = await find.existsByCssSelector('[label="road_map"]');
-          expect(roadMapExists).to.be(true);
+          const options = await find.allByCssSelector('#wmsOptionsSelectTmsLayer > option');
+          const optionsText = await Promise.all(options.map(async (optionElement) => {
+            return await optionElement.getVisibleText();
+          }));
```

Comment:
I think using innerHTML is more stable/fast way to get a text here:

```suggestion:-4
          const $ = await selectField.parseDomContent();
          const optionsText = $('option').toArray().map(option => $(option).text());
```

Reply:
Thanks!
Also changed the `PageObjects.visualize.setSelectByOptionText` due to suggestion